backend/contracts/OasisReward.sol

```diff
@@ -1,19 +1,32 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import {ERC721, ERC721Enumerable} from "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
-import {Base64} from "@openzeppelin/contracts/utils/Base64.sol";
-
+import {
+    ERC721, ERC721Enumerable
+} from "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import {
+    Base64
+} from "@openzeppelin/contracts/utils/Base64.sol";
+
+/**
+ * @title OasisReward
+ * @notice An ERC721 contract that allows minting of NFT rewards for 
+    Oasis Quizzes.
```

Comment:
I think multi-line natspec starts with * in multiline /** comments.

```suggestion
 *  Oasis Quizzes.
```

Reply:
Fixed.